stackhpc/stackhpc-kayobe-config
# 2023.1: zed merge

- CloudKitty fix for ES->OS migration
- Document the manual process to delete a broken LB
- docs: a couple of bits for the RL9 migration
- OVN: Disable distributed floating IPs by default
- Update Consul and Vault to latest MPL releases
- Handle undefined Ansible variable
- Fix growroot playbook for disks ending with p
- Add release note
- Build Neutron from StackHPC fork
- Bump Neutron tag to fix multiple bugs
- Bump Ceph images to latest stable releases
- Check if repos backup exists in kolla footer
- Restore flavor_id on openstack_nova_server_status metrics
- Still check for stackhpc repos in footer
- Apply suggestions from code review
- Fix bash in footer
- Fix libvirtd memory leak in CentOS Stream 8
- Automatic openstack release for kolla sources
- docs: add more notable changes and known issues for Zed
- Add cloudkitty tag
- New neutron tag
- docs: Fix typo
- Fix cloudkitty_tag
- Remove invalid reno for zed
- Add Neutron tag
- Fix Ceph reno for Zed
